src/v2/Apps/Home/Components/HomeFeaturedGalleriesRail.tsx

```diff
@@ -150,9 +191,11 @@ export const HomeFeaturedGalleriesRailFragmentContainer = createFragmentContaine
               ... on Profile {
                 ...FollowProfileButton_profile
                 internalID
+                isFollowed
                 name
                 slug
                 href
+                # following
```

Comment:
?

Reply:
ah, this was me guessing at a field -- it's actually `isFollowed`